nautobot/extras/models/groups.py

```diff
@@ -67,14 +73,23 @@ class DynamicGroup(OrganizationalModel):
     # to dynamically change the widget if we decide we do want to support this field.
     #
     # Type: tuple
-    exclude_filter_fields = ("q", "cr", "cf", "comments")  # Must be a tuple
+    exclude_filter_fields = ("q", "cf_", "cr_", "cpf_", "comments")  # Must be a tuple
```

Comment:
It works, but currently it's being excluded because of the issue with the form-field widgets not being correctly applied and it mucking up the UI. It's a solvable problem, just not yet in scope.

Reply:
Then again, maybe with some recent changes they work now? Can comment this out and try...